YACReader/viewer.cpp

```diff
@@ -634,20 +634,20 @@ void Viewer::keyPressEvent(QKeyEvent *event)
         QAbstractScrollArea::keyPressEvent(event);
 }
 
-void Viewer::moveAction(QKeySequence key)
+void Viewer::moveAction(QKeySequence *key)
```

Comment:
Did you try declaring this method as `void Viewer::moveAction(const QKeySequence &key)`? then you can invoke it as `moveAction(key)` and there is no need for indirection in the method's code. I think we can merge the PR after changing it.